parts/windows/windowscontainerdfunc.ps1

```diff
@@ -37,9 +37,8 @@ function RegisterContainerDService {
   if ($null -eq $svc) {
     Set-ExitCode -ExitCode $global:WINDOWS_CSE_ERROR_CONTAINERD_NOT_INSTALLED -ErrorMessage "containerd.exe did not get installed as a service correctly."
   }
-  Start-Service containerd
   if ($svc.Status -ne "Running") {
-    Set-ExitCode -ExitCode $global:WINDOWS_CSE_ERROR_CONTAINERD_NOT_RUNNING -ErrorMessage "containerd service is not running"
+    Start-Service containerd

```

Comment:
Do you think that start-service will always succeed? Should we need to check it again?
I think that we should not remove the error when containerd is not running.

Reply:
Added an exception check.